lib/spacesuit/api_message.ex

```diff
@@ -5,7 +5,7 @@
 # Just does JSON for now
 defmodule Spacesuit.ApiMessage do
   @derive [Poison.Encoder]
-  defstruct [:status, :message]
+  defstruct [:errorCode, :errorMessage]
```

Comment:
No, generally not. In most cases it's more descriptive, e.g. INVALID_TOKEN, PERMISSION_DENIED (if the permission exists but the account doesn't match), UNVERIFIED_USER, ... but could also just be UNAUTHORIZED or FORBIDDEN